contracts/ZeroCollateral.sol

```diff
@@ -343,7 +343,19 @@ contract ZeroCollateralMain is ERC20Detailed, ERC20, MinterRole, Chainlink {
         }
     }
 
+    function getCollateralNeeded(uint256 amountBorrow) view public returns(uint256){
+        uint256 poolContributions = (borrowerAccounts[msg.sender].amountPaidRedemptionPool);
+        uint256 amountBorrowDecimals = amountBorrow.mul(10**6); // convert to DAI decimals
```

Comment:
Maybe I've misunderstood, but I believe DAI has 18 decimals not 6. The contract has a constant called `DAI_DECIMALS` storing `10**18` for this purpose

Reply:
You're right. The DAI has 18 decimals. This function was added again because the contract didn't compile, but the full contract will be reviewed to clean not needed functions, and issues like this.